README.md

```diff
@@ -40,6 +40,6 @@ docker build -t vole-app-image .
 docker run --rm -p 9020:80 --name vole-app vole-app-image
 ```
 
-This will create a new docker image called `vole-app` and run it on port 9020. You can access the viewer by navigating to http://localhost:9020 in your browser.
+This will create a new docker image called `vole-app` and run it on port 9020. You can access the viewer by navigating to [http://localhost:9020](http://localhost:9020) in your browser.
```

Comment:
just curious: does this work as a link without the identical URL specified in `( )` after it?

Reply:
It does! It also works as a plaintext link without the `[]()` notation. The markdown linter I have doesn't like empty links though.